hasura.planx.uk/metadata/tables.yaml

```diff
@@ -2661,36 +2710,49 @@
     - role: demoUser
       permission:
         columns:
-          - created_at
-          - email
-          - first_name
           - id
-          - is_platform_admin
+          - first_name
           - last_name
+          - created_at
           - updated_at
-        filter: {}
+          - is_platform_admin
+          - is_staging_only
+          - email
+        filter:
+          created_flows:
+            creator_id:
+              _eq: '"x-hasura-user-id"'
     - role: platformAdmin
       permission:
         columns:
           - id
           - first_name
           - last_name
-          - email
           - created_at
           - updated_at
           - is_platform_admin
+          - is_staging_only
+          - email
         filter: {}
     - role: teamEditor
       permission:
         columns:
-          - created_at
-          - email
-          - first_name
           - id
-          - is_platform_admin
+          - first_name
           - last_name
+          - created_at
           - updated_at
-        filter: {}
+          - is_platform_admin
+          - is_staging_only
+          - email
+        filter:
+          teams:
+            _and:
+              - user_id:
+                  _eq: '"x-hasura-user-id"'
```

Comment:
```suggestion
                  _eq: x-hasura-user-id
```

Hasura sneakily wraps their input in quotation marks - this has caught me out before.

![image](https://github.com/user-attachments/assets/3fcb8f9a-574e-4b0c-9415-45423b0c0d28)

Reply:
Sneaky indeed - thanks for catching these - didn't even think to check the autogenerated syntax !